upp-aurora-postgres-provisioner/config/vars/prod-us.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2034
+
+set -euo pipefail
+IFS=$'\n\t'
+
+ENVIRONMENT_DESC="Prod"
+RESOURCES_PREFIX="upp"
+
+AWS_REGION="us-east-1"
+VPC_ID="vpc-4bc03f32"
+CLUSTER_CIDR="10.168.64.0/18"
+CLUSTER_CIDR_DESC="FT-Content-Platform-Prod vpc-4bc03f32 CIDR"
+RESOURCES_SG_ID="sg-a0e39ade"
+RDS_SUBNETS="subnet-0ef54f5f927721d15, subnet-0c18f15e56e0490c6, subnet-0ef8d475f498f5928"
+
+ENVIRONMENT_TAG="p"
+SYSTEM_CODE_TAG="upp"
+TEAM_DL_TAG="universal.publishing.platform@ft.com"
+
+CF_STACK_NAME="${RESOURCES_PREFIX}-${CF_TEMPLATE}-${ENVIRONMENT_NAME}"
```

Comment:
**question:** I'm wondering whether we are in situation of having the same `CF_STACK_NAME` name for provisioning/deleting EU and US DBs (which I think could lead to problems). I'm not 100% sure here.
So for example for both EU and US the CloudFormation stack will be `upp-enriched-content-rds-prod`?

Reply:
Yes, they will be the same and the `REGION` envvar would be used to disambiguate between the stacks. 

Initially I changed the provisioner in such a way that the stack name(and the db name respectively) was in the form `upp-enriched-content-rds-<env>-<region>` e.g. `upp-enriched-content-rds-staging-us`, but then I saw that none of our RDSs has region in its name and also the one in EU is already provisioned without the region name.